internal/oci/ecr/ecr.go

```diff
@@ -5,50 +5,122 @@ import (
 	"encoding/base64"
```

Comment:
Idk the pattern of project you're using but I'd suggest to keep the name without the _ECR _ prefix as the package is already `ecr`.
* Client
* PublicClient
* PrivateClient

Reply:
good catch. thank you